documentation/DoD.asciidoc

```diff
@@ -22,7 +22,7 @@ However, chances to get your change merged quickly are higher if you address all
 * [ ] You followed all coding conventions (indent with 2 spaces, no tabs, functions start with `do` prefix, variables are enclosed with curly braces, variables in functions are declared with `local`, etc.)
 * [ ] You have already added the issue implemented by your PR in https://github.com/devonfw/ide/blob/master/CHANGELOG.asciidoc[CHANGELOG.asciidoc] to the next open release (see milestones or https://github.com/devonfw/ide/blob/master/.mvn/maven.config[maven.config]). If there is no issue for your PR consider creating it or directly link the PR itself. Please note that the CHANGELOG shall only reflect public changes relevant for end-users. So e.g. if we implement a story and then add another PR as bugfix or improvement to the same story, we do not need to document this in the CHANGELOG to avoid spam and confusion.
 * [ ] In case your PR adds a new tool `«tool»` as commandlet there are the following additional checks:
-** [ ] The tool is already available as folder `«tool»` in https://github.com/devonfw/ide-mirrors[ide-mirrors] including `available-versions`.
+** [ ] A new urlUpdater named `«tool»` has been added to a a folder named `«tool»` in https://github.com/devonfw/ide/tree/master/url-updater/src/main/java/com/devonfw/tools/ide/url/updater[updater] that receives response bodies from URLs, updates tool versions, and checks if download URLs work.
```

Comment:
@CREITZ25 just created a PR that will change the location.
Instead of
https://github.com/devonfw/ide/tree/master/url-updater/src/main/java/com/devonfw/tools/ide/url/updater
you need to use
https://github.com/devonfw/ide/tree/master/ide/src/main/java/com/devonfw/tools/ide/url/updater

However, I can now merge this and @CREITZ25 can include the documentation change into his PR.